sparkle/src/icons/solid/RobotShared.tsx

```diff
@@ -1,24 +0,0 @@
-import type { SVGProps } from "react";
```

Comment:
This one is used in production 🙃, make sure to refactor once you open a branch on `main` to use this new version.